File: cslibrary4a/src/main/java/com/csl/cslibrary4a/ConnectorData.java
package com.csl.cslibrary4a;

public class ConnectorData {
    public enum ConnectorTypes {
        RFID, BARCODE, NOTIFICATION, SILICONLAB, BLUETOOTH, OTHER
    }
    public ConnectorTypes connectorTypes;
    public byte[] dataValues;
    public boolean invalidSequence;
    public long milliseconds;
}

File: cslibrary4a/src/main/java/com/csl/cslibrary4a/ConnectorData.kt
package com.csl.cslibrary4a

class ConnectorData {
    enum class ConnectorTypes {
        RFID, BARCODE, NOTIFICATION, SILICONLAB, BLUETOOTH, OTHER
    }

    @JvmField
    var connectorTypes: ConnectorTypes? = null
    @JvmField
    var dataValues: ByteArray? = null

    @JvmField
    var invalidSequence: Boolean = false
    @JvmField
    var milliseconds: Long = 0
}
